pyaedt/modeler/cad/Modeler.py

```diff
@@ -1134,6 +1207,388 @@ def _attributes(self):
         return coordinateSystemAttributes
 
 
+class ObjectCoordinateSystem(BaseCoordinateSystem, object):
+    """Manages object coordinate system data and execution.
+
+    Parameters
+    ----------
+    modeler :
+        Inherited parent object.
+    props : dict, optional
+        Dictionary of properties. The default is ``None``.
+    name : optional
+        The default is ``None``.
+    entity_id : int
+        Id of the entity object where the Object Coordinate System is anchored.
+
+    """
+
+    def __init__(self, modeler, props=None, name=None, entity_id=None):
+        BaseCoordinateSystem.__init__(self, modeler, name)
+        self.entity_id = entity_id
+        self._props = None
+        if props:
+            self._props = CsProps(self, props)
+            try:  # pragma: no cover
+                if "KernelVersion" in self.props:
+                    del self.props["KernelVersion"]
+            except:
+                pass
+        self._ref_cs = None
+
+    @property
+    def ref_cs(self):
+        """Reference coordinate system getter and setter.
```

Comment:
```suggestion
        """Reference coordinate system.
```
According to Alex K, we didn't need the getter and setter indication.